### ISSUE TITLE
bypass cloudflare ddos

### ISSUE THREAD

**Issue by abuhamsa** [abuhamsa/rogers2xmltv#18]
i'll guess adding a header agent could help.
https://devdreamz.com/question/577755-scraping-news-headlines-using-requests-or-selenium-cant-return-data

**Comment by abuhamsa** [comment]
```
<div class="attribution">
DDoS protection by <a rel="noopener noreferrer" href="https://www.cloudflare.com/5xx-error-landing/" target="_blank">Cloudflare</a>

<br />
<span class="ray_id">Ray ID: <code>70310229f9a732ac</code></span>
</div>
```

**Comment by abuhamsa** [comment]
fixed with #19
